snuba/migrations/snuba_migrations/transactions/0011_transactions_add_span_op_breakdowns.py

```diff
@@ -0,0 +1,64 @@
+from typing import Sequence
+
+from snuba.clickhouse.columns import Column, Float, Nested, String
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations
+from snuba.migrations.columns import MigrationModifiers as Modifiers
+
+
+class Migration(migration.ClickhouseNodeMigration):
+    """
+    Adds the span_op_breakdowns nested column
+    """
+
+    blocking = False
+
+    def forwards_local(self) -> Sequence[operations.SqlOperation]:
+        return [
+            operations.AddColumn(
+                storage_set=StorageSetKey.TRANSACTIONS,
+                table_name="transactions_local",
+                column=Column(
+                    "span_op_breakdowns",
+                    Nested(
+                        [
+                            ("key", String(Modifiers(low_cardinality=True))),
+                            ("value", Float(64)),
+                        ]
+                    ),
+                ),
+                after="_contexts_flattened",
```

Comment:
I would make this after `measurements`. Small nit.

Reply:
👍 for this, we keep the order of columns the same in the physical and Snuba schemas by convention

Reply:
Good catch.